doc/articles/guides/localization.md

```diff
@@ -93,7 +93,8 @@ This guide will walk you through the necessary steps to localize an Uno Platform
     -|-|-
     MainPage_IntroText.Text|Hello Uno|Bonjour Uno
     MainPage_CodeBehindString|String from code-behind|Texte provenant du code-behind
```

Comment:
i think docfx is having trouble telling the table apart from the text above.
can you add an empty line before this table?